guide-gen/guide-configs/concepts/cloud.json

```diff
@@ -0,0 +1,47 @@
+{
+  "file_name": "index.md",
+  "id": "index",
+  "file_dir": "cloud",
+  "title": "Temporal Cloud documentation",
+  "sidebar_label": "Temporal Cloud",
+  "description": "Teporal Cloud documentation explanation and usage information.",
+  "toc_max_heading_level": 4,
+  "add_tabs_support": false,
+  "h2_sections": [
+    {
+      "header": "none",
+      "h3_sections": [
+        {
+          "header": "none",
+          "type": "explanation",
+          "path": "concepts/what-is-temporal-cloud"
+        }
+      ]
+    },
+    {
+      "header": "Cloud Namespace",
+      "h3_sections": [
+        {
+          "header": "none",
+          "type": "explanation",
+          "path": "concepts/what-is-a-cloud-namespace-name"
+        },
+        {
+          "header": "Cloud Namespace Id",
+          "type": "explanation",
+          "path": "concepts/what-is-a-cloud-namespace-id"
+        }
```

Comment:
I can certainly see the argument for placing this term under Cloud Namespace, but doing so creates a single bullet at that level. I don't have any other "What is" topics planned that could be peers of Cloud Namespace Id, so I'm inclined to raise this section to the same level as the other two terms.